build_tools/cirrus/arm_wheel.yml

```diff
@@ -68,6 +83,9 @@ linux_arm64_wheel_task:
         CIBW_TEST_SKIP: "*_aarch64"
     - env:
         CIBW_BUILD: cp311-manylinux_aarch64
+        CIBW_TEST_SKIP: "*_aarch64"
+    - env:
+        CIBW_BUILD: cp312-manylinux_aarch64
```

Comment:
Can we continue to test for 3.11 and leave 3.12 untested? This way, when we merge this, we are still testing on the latest **released** version of Python.